splunk-on-ubuntu/azuredeploy.json

```diff
@@ -167,9 +198,12 @@
     "sharedTemplateUrl": "[concat(variables('templateBaseUrl'), 'nested/', 'shared-resources.json')]",
     "virtualNetworkTemplateUrl": "[concat(variables('templateBaseUrl'), 'nested/', 'vnet-', parameters('virtualNetworkNewOrExisting'), '.json')]",
     "emptyTemplateUrl": "[concat(variables('templateBaseUrl'), 'nested/', 'empty-resources.json')]",
+    "nodesPerStorageAccount": 4,
     "tshirtSizeStandalone": {
-      "vmSize": "[parameters('vmSize')]",
-      "vmSizeClusterMaster": "Standard_D3",
+      "standaloneVmSize": "[parameters('standaloneVmSize')]",
+      "clusterMasterVmSize": "",
```

Comment:
These properties (e.g `clusterMasterVMSize`) are null in `tshirtSizeStandalone` because they are not applicable for the standalone case. They still need to be enumerated because they're presented as input for nested templates - which would point to `empty-resources.json` in the standalone case.